Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -156,7 +157,7 @@ import PayPalCheckout
                     },
                     environment: order.environment
                 )
-
+                payPalNativeConfig.authConfig.userEmail = userAuthenticationEmail
```

Comment:
We have access to the request here, we shouldn't need to add this to the method signature
```suggestion
                payPalNativeConfig.authConfig.userEmail = request.userAuthenticationEmail
```

Reply:
We have access to the request but `userAuthenticationEmail` is not initialized in `BTPayPalRequest` but rather `BTPayPalCheckoutRequest`. Do you think we should initialize it in `BTPayPalRequest` instead?

Reply:
Ah gotcha - we can add it to `BTPayPalRequest` if we've confirmed it works for all of the flows (PayPal Web Checkout, PayPal Web Vault; MXO Vault, MXO Checkout). I am guessing this has not been confirmed since we are setting it here in `payPalNativeConfig.authConfig` but for `BTPayPalCheckoutRequest` and `BTPayPalVaultRequest` we aren't doing anything with this parameter at all.

Also do we need to do a release on the MXO side to start using this?

Reply:
Yes we need to a release on the MXO side so this is on hold and I am testing locally

Reply:
Does this need a release before being added to the web based flows as well (PayPal Web Checkout and PayPal Web Vault)? We won't want to add this to `BTPayPalVaultRequest` or `BTPayPalCheckoutRequest` if it doesn't work for those flows.